Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -87,7 +87,12 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
 
     @objc func universalLinkFlow(_ sender: UIButton) {
         // TODO: implement in a future PR - used here so we don't have to remove lazy instantiation
-        let request = BTPayPalVaultRequest(enablePayPalAppSwitch: true)
+        // TODO: replace URL with https://braintree-ios-demo.fly.dev/braintree-payments
```

Comment:
Curious why we can't use: "https://braintree-ios-demo.fly.dev/braintree-payments"

Reply:
The gateway has not yet allowlisted our URL for this flow, so we are using a URL that is included in their allowlist currently.